### PR TITLE
Jobs page improved

### PR DESCRIPTION
**Pull request by julien51**

### CODE REVIEW

File: unlock-app/src/components/pages/Jobs.js [hunk 1]
```diff
@@ -7,31 +8,114 @@ export const Jobs = () => (
...
+      <Headline>We thrive to provide an environment where you can do your best work. This includes (but is not limited to):</Headline>
+      <ShortColumn>
+
+        <UnorderedList>
+          <ListItem>Comprehensive health care (including dental and vision) for you 100% paid by Unlock (75% for your dependents). </ListItem>
+          <ListItem>Unlimited personal and vacation days.</ListItem>
+          <ListItem>Learning and development budget for each employee.</ListItem>
+          <ListItem>Monthly wellness stipend that can be used for a gym membership, nutrition counseling, yoga or meditation classes, or any other wellness activity of your choice. </ListItem>
+          <ListItem>Your choice of technical setup (laptop, monitors and software licenses).</ListItem>
+          <ListItem>Catered lunch ever day in the office.</ListItem>
```
Comment (benwerd) [inline unlock-app/src/components/pages/Jobs.js]: Typo: catered lunch every day.

File: unlock-app/src/components/pages/Jobs.js [hunk 2]
```diff
@@ -7,31 +8,114 @@ export const Jobs = () => (
...
+      <Section anchor="full-stack-engineer">
+        <Title>Full Stack Engineer</Title>
+        <JobDetails>New York — Engineering — Full Time</JobDetails>
+        <SubTitle>About You</SubTitle>
+
+        <Paragraph>
+          Do you think that the web&#39;s <a href="https://www.theatlantic.com/technology/archive/2014/08/advertising-is-the-internets-original-sin/376041/">original sin</a> is its lack of business model? Do you think that individuals, democracies and the web, deserve better than click-bait, information overload or fake news?
+        </Paragraph>
+        <Paragraph>
+          Are you excited about empowering creators by building the tools and interfaces which will let them monetize their creations without gate keepers?
```
Comment (benwerd) [inline unlock-app/src/components/pages/Jobs.js]: Gatekeepers should be one word here

File: unlock-app/src/components/pages/Jobs.js [hunk 3]
```diff
@@ -7,31 +8,114 @@ export const Jobs = () => (
...
+          <ListItem>Extensive communication skills to ensure that efficient collaboration with other team members</ListItem>
+          <ListItem>Be a steward and influencer of our early engineering culture</ListItem>
+          <ListItem>In NYC or willing to relocate</ListItem>
+        </UnorderedList>
+
+        <SubTitle>Great to have</SubTitle>
+        <UnorderedList>
+          <ListItem>Managed small to mid size teams (including hiring and mentoring)</ListItem>
+          <ListItem>Experience working on open source projects</ListItem>
+          <ListItem>Passion of cryptography and cyber security</ListItem>
```
Comment (benwerd) [inline unlock-app/src/components/pages/Jobs.js]: Passion _for_

File: unlock-app/src/components/pages/Jobs.js [hunk 4]
```diff
@@ -7,31 +8,114 @@ export const Jobs = () => (
...
+          <ListItem>Interest for user experience and visual arts</ListItem>
```
Comment (benwerd) [inline unlock-app/src/components/pages/Jobs.js]: Interest _in_

Comment (benwerd) [reply unlock-app/src/components/pages/Jobs.js]: _the_ visual arts

File: unlock-app/src/components/pages/Jobs.js [hunk 5]
```diff
@@ -7,31 +8,114 @@ export const Jobs = () => (
...
+        </UnorderedList>
+
+        <SubTitle>Compensation : $130K – $160K, stock options</SubTitle>
+      </Section>
+
+      <Section>
+        <Title>Applications</Title>
+        <Paragraph>
+          If you are interested in applying for a position at Unlock, please send an email containing your resume, Github, and Linkedin to <a href="mailto:julien@unlock-protocol.com">julien@unlock-protocol.com</a>, and reach out to Julien in our <a href="https://t.me/unlockprotocol">Telegram group</a>!).
```
Comment (benwerd) [inline unlock-app/src/components/pages/Jobs.js]: LinkedIn (capital I)

**Comment by julien51** [comment]
Thanks! all fixed!

**Pull request merged**